src/sagemaker/lineage/artifact.py

```diff
@@ -522,3 +584,75 @@ def endpoint_contexts(
         for vertex in query_result.vertices:
             endpoint_contexts.append(vertex.to_lineage_object())
         return endpoint_contexts
+
+    def get_upstream_datasets(
```

Comment:
Let's remove `get_` from all of these for consistency

Reply:
> Let's remove `get_` from all of these for consistency

removed